### DISCUSSION TITLE
[FR] How. does it sync cross devices?

### DISCUSSION THREAD

**Discussion by lucasjinreal** [AppFlowy-IO/AppFlowy#540] [General help wanted]
currently now account, how should users sync docs cross devices?

**Comment by lucasjinreal** [comment]
@annieappflowy Would u please answer my question?

**Comment by annieappflowy** [comment]
Hi @jinfagang we don't support syncing across devices yet. This is something we will start developing from v0.0.5
Thanks for checking though. It would be great if you can provide more info on your use cases for this feature.

**Reply by JoelMon** [reply]
I for one would need Android support like Obsidian has. Obsidian desktop on Linux and on Android is so seamless. ♥️ that.

**Comment by annieappflowy** [comment]
Hey guys, we've decided to build a flowy editor first before working on this feature as we consider AppFlowy's editing experience as our top priority at the moment. Please check this 6-month [roadmap](https://github.com/AppFlowy-IO/AppFlowy/discussions/565) for more details. I'm converting this into a [FR] issue for us to better log the requirements and will come back when in development. Meanwhile, if you have more insights or requirements related to syncing, please feel free to share here. Thank you!

**Reply by DominiqueSite** [reply]
I would love for AppFlowy to have an offline-only mode. If I do not have a use case where I want to collaborate with other people, AppFlowy making outward connections is really more of an anti-feature (at least if I get my updates via e.g. Flathub).
For syncing across devices, most people already have their own preferred file-sharing solution ([Syncthing](https://github.com/syncthing/syncthing) in my case, maybe [Seafile](https://github.com/haiwen/seafile) or Dropbox for others) that they use to sync files across their devices.
In my use case (without the need for collaboration), I would simply sync a local directory with AppFlowys configuration and content files. If the names of the files are valid across all supported operating systems, the sync is smoothly handled by another program on the local device and there is no need for any server "messing" with my data (altough AppFlowy would need to be aware of local changes to files while running).
As for the collaboration use case, I am sure you will figure out smart ways to do that too, with or without a server.

**Reply by annieappflowy** [reply]
Big thanks for sharing your use cases @DominiqueSite I suppose I totally get your point because I use Google Drive to sync my Obsidian data across devices. We'll consider supporting popular file-sharing solutions such as Syncthing when designing the custom data storage feature. You can keep an eye out for v0.0.5+, and please provide more feedback along the way!
